### ISSUE TITLE
0.12.0 regression? Flex subviews aren't the same height when superview doesn't have height

### ISSUE THREAD

**Issue by Sidnicious** [facebook/react-native#3429]
Possibly related: #1472

When a view has `flexDirection: 'row'` and no explicit height and its child views have `flex: 1`, like this:

```jsx
<View style={{marginTop: 20, flexDirection: 'row'}}>
  <View style={{flex: 1, backgroundColor: 'green'}}><Text>Quite a lot of content oh god why so much content</Text></View>
  <View style={{flex: 1, backgroundColor: 'blue'}}><Text>Some content</Text></View>
</View>
```

…the child views should be the same height:

<img width="317" alt="" src="https://cloud.githubusercontent.com/assets/111870/10500735/624c4b22-72a8-11e5-8beb-fe0546b827c2.png">

…this works fine [on rnplay](https://rnplay.org/apps/VyQH9Q), which uses 0.11, but if I paste the same code into an 0.12.0 project, this happens:

<img width="487" alt="" src="https://cloud.githubusercontent.com/assets/111870/10500690/25df2f56-72a8-11e5-8fa5-ed0288601190.png">

**Comment by cosmith** [comment]
This seems to be fixed in 0.13

![screen shot 2015-10-29 at 12 24 51 pm](https://cloud.githubusercontent.com/assets/1194749/10817119/230011f6-7e38-11e5-82ea-4793a7369a59.png)

**Comment by Sidnicious** [comment]
Confirmed, thanks for checking!
